lib/decorators/pick-allowable-fields.ts

```diff
@@ -1,14 +1,14 @@
-import { describe, Schema } from 'joi';
+import { describe, Schema } from 'joi-browser';
```

Comment:
Does this work when you install beneficiary-reporting-validation into the api?

Reply:
Yea, i re-tested the api as well. Wouldn't hurt for someone else to try it though to make sure